doc/source/advanced.rst

```diff
@@ -260,6 +272,22 @@ Passing a list of labels or tuples works similar to reindexing:
 
    df.loc[[('bar', 'two'), ('qux', 'one')]]
 
+.. warning::
+
+   It is important to note that tuples and lists are not treated identically
+   in pandas.
```

Comment:
I think the wording is OK, but I would add another sentence stating the different role (multi-level key vs. list of keys).